q2_feature_classifier/_vsearch.py

```diff
@@ -0,0 +1,64 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import pandas as pd
+from ._consensus_assignment import (_consensus_assignments,
+                                    _get_default_unassignable_label,
+                                    _validate_params)
+from q2_types.feature_data import (
+    FeatureData, Taxonomy, Sequence, DNAFASTAFormat)
+from .plugin_setup import plugin
+from qiime2.plugin import Int, Str, Float
```

Comment:
add `Choices, Range` to this last import (see my change below which will use these).